config/locales/en.yml

```
@@ -391,570 +408,624 @@ en:
-            show_to_nations:
-              - Scotland
-          - id: "0022"
-            text: "What to do if you cannot pay your bills or mortgage (Citizens Advice)"
-            href: "https://www.citizensadvice.org.uk/wales/debt-and-money/if-you-cant-pay-your-bills-because-of-coronavirus/"
-            show_to_nations:
-              - Wales
-          - id: "0023"
-            text: Get help from Citizens Advice
-            href: "https://www.citizensadvice.org.uk/about-us/northern-ireland/"
-            show_to_nations:
-              - Northern Ireland
-          - id: "0024"
-            text: "Get help from the housing and debt helpline for Northern Ireland (Housing Rights)"
-            href: "https://www.housingrights.org.uk/"
-            show_to_nations:
-              - Northern Ireland
-          - id: "0025"
-            text: "What to do if you’re finding it hard to pay rent (Welsh Government)"
-            href: "https://gov.wales/coronavirus-covid-19-guidance-for-tenants-in-the-private-rented-sector-html"
-            show_to_nations:
-              - Wales
-          - id: "0026"
-            text: "Find out about your rights if you have a private landlord (mygov.scot)"
-            href: "https://www.mygov.scot/private-rental-rights/"
-            show_to_nations:
-              - Scotland
-          - id: "0027"
-            text: "Find out about your rights if you have a social landlord (mygov.scot)"
-            href: "https://www.mygov.scot/social-rental-rights/"
-            show_to_nations:
-              - Scotland
+        - id: '0016'
+          text: Your rights if you’re finding it hard to pay rent
+          href: https://www.gov.uk/government/publications/covid-19-and-renting-guidance-for-landlords-tenants-and-local-authorities
+        - id: '0017'
+          text: What to do if you cannot pay your rent or mortgage (Shelter)
+          href: https://england.shelter.org.uk/housing_advice/coronavirus
+          show_to_nations:
+          - England
+        - id: '0018'
+          text: What to do if you cannot pay your rent or mortgage (Shelter Scotland)
+          href: https://scotland.shelter.org.uk/get_advice/scottish_housing_advice_coronavirus_COVID_19
+          show_to_nations:
+          - Scotland
+        - id: '0019'
+          text: What to do if you cannot pay your rent or mortgage (Shelter Cymru)
+          href: https://sheltercymru.org.uk/get-advice/coronavirus/
+          show_to_nations:
+          - Wales
+        - id: '0020'
+          text: What to do if you cannot pay your bills or mortgage (Citizens Advice)
+          href: https://www.citizensadvice.org.uk/debt-and-money/if-you-cant-pay-your-bills-because-of-coronavirus/
+          show_to_nations:
+          - England
+        - id: '0021'
+          text: What to do if you cannot pay your bills or mortgage (Citizens Advice
+            Scotland)
```

Multiple lines again:

```suggestion
          text: |
            What to do if you cannot pay your bills or mortgage (Citizens Advice
            Scotland)
```

---

@TODO confirmation screenshot on the way

See here for why i think we shouldn't change these: https://github.com/alphagov/govuk-coronavirus-find-support/pull/222/commits/b0c5b0aafdd33f6b983cf66d96c9d34c0e9bfcf6#r430281216